compiler/damlc/tests/daml-test-files/DefaultMethods.daml

```diff
@@ -31,3 +31,11 @@ class Id a where
   id = \x -> x
 
 instance Id Int where
+
+-- | Default implementation with a separate type signature for the default method.
+class MyShow t where
```

Comment:
I believe this PR also fixes the `(+), (-) : a -> a -> a` case right? If so, can we get a test for that as well?

Reply:
I think that was already fixed, but a test case can't hurt :)